JobParseAndNormalize.md

```diff
@@ -0,0 +1,353 @@
+Job Parse and Normalize API
+====================
+
+####Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+##Summary
+
+The Job Parse and Normalize (JPAN) service parses a Base64-encoded job posting and further enriches the parsed data with the following normalizations and classifications.  
+
+ #####Enrichments:
+ - Job Title Classifications
+    - [Carotene](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - [ONET](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - Textkernel
+ - [Normalized Company Name](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/CompanyNormalization.md)  <sup>(optional)</sup>
+ - Skills Extraction
+    - [Skills 4.0](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Skills.md) <sup>(optional)</sup>
```

Comment:
Yeah I agree it kind of sucks, but we call it "4.0" in the response because we decided we didn't want to call it "skill": "careerbuilder". 

```
      "skills": {
        "textkernel": [
          {
            "name": string,
            "skill_code": integer,
            "required": boolean,
            "skill_group": string
          },
          ...
        ],
        "4.0": [
          {
            "skilldid": string,
            "normalized_term": string,
            "confidence": string,
            "type": string
          },
          ...
        ]
      },
```

Basically, I need to call it something besides ```skills``` then. Thoughts?